srcpkgs/imv/template

```diff
@@ -18,5 +18,8 @@ checksum=8c2f1baa4dce8bf1f6d1fb9dea8cf1da09fdf2c6de0f7030e91714df2ebcdf50
 conflicts="renameutils>=0"
 
 post_install() {
+	# copy the man page so it isn't clobbered by the one from renameutils
+	vman "${DESTDIR}/usr/share/man/man1/imv.1" imv-x11.1
+	ln -sf /usr/share/man/man1/imv-x11.1 "${DESTDIR}/usr/share/man/man1/imv-wayland.1"
```

Comment:
Do we need those 2 manpages?
Is `vman` a `cp` or `mv`? If it's the latter, I guess no more conflict with `renameutils`?

Reply:
`vman` is a `cp` afaik. And the main conflict is because of the `/usr/bin/imv` binaries in both. This doesn't fix that conflict, just helps us with being able to show a man page for `imv` the image viewer in <man.voidlinux.org>.

Upstream has an open issue about renaming, but I don't know what they decided.